examples/angular/src/pages/ui/primitive-poc/text-field/text-field.component.html

```diff
@@ -0,0 +1,44 @@
+<amplify-poc-text-field
+  labelHidden="true"
+  describedBy="custom-aria-describedBy"
+>
+  <p amplify-label>Last name</p>
+  <p amplify-description-text>Enter Description</p>
+  <input amplify-input placeholder="Baggins" />
+  <p amplify-error>There is an error</p>
+</amplify-poc-text-field>
+<br />
+<br />
+<br />
+<br />
+
+<amplify-poc-text-field hasError="true" describedBy="custom-aria-describedBy">
+  <p amplify-label>Enter Label</p>
+  <p amplify-description-text>Enter Description</p>
+  <input amplify-input placeholder="Baggins" isDisabled="true" />
+  <p amplify-error>There is an error</p>
+</amplify-poc-text-field>
+
+<br />
+<br />
+<br />
+<br />
+
+<amplify-poc-text-field hasError="true" describedBy="custom-aria-describedBy">
```

Comment:
Shouldn't this be
```suggestion
<amplify-poc-text-field [hasError]="true" describedBy="custom-aria-describedBy">
```
because it's a boolean?

Reply:
Changed.